ale/drivers/base.py

```diff
@@ -1,13 +1,11 @@
-from abc import ABC, abstractmethod
-
 from dateutil import parser
 import numpy as np
 import pvl
 import spiceypy as spice
 
 from ale.drivers import distortion
 
-class Base(ABC):
+class Base:
```

Comment:
This should or should not be an abstract base? @Kelvinrr?

Reply:
Technically no, we moved away from using ABCs to enforce keys and instead used the other approach of using a required key set.